manim/mobject/svg/text_mobject.py

```diff
@@ -1064,3 +1065,205 @@ def text2svg(self):
             offset_x += pangocffi.units_to_double(layout.get_size()[0])
         surface.finish()
         return file_name
+
+
+# FIXME: documentation
+# <b>, <i>, <tt>, <big>, <small>, <s>, <sub>, <sup>, <u>
+# <span font_family="...">
+# <span underline="..."> single,double,error
+# <color col="..."> with #xxxxxx or manim constant
+# <gradient from="..." to="...">
+class MarkupText(Text):
```

Comment:
Wondering why it is inheriting from `Text`? Shouldn't it be SVGMobject?

Also, I feel like SVGMobject isn't that good, should rewrite all these to use directly inherit from `VMobject`.

Reply:
I use `remove_last_M` from `Text`. I could, however, copy that method over to the new class. That's probably the best, in case `MarkupText` replaces the current `Text` eventually.